### PR TITLE
Pntn 218 table rules overview

### PR DESCRIPTION
**Pull request by ctizen**
https://pantheon.myjetbrains.com/youtrack/issue/PNTN-218

Also update twitter bootstrap to 4.0.0 stable.

### CODE REVIEW

File: Rheda/src/helpers/Config.php [hunk 1]
```diff
@@ -49,6 +49,57 @@ class Config
...
+            'eventStatHost' => null, // not intended to be shown in rule overview
+            'redZone' => _t('Red zone duration'),
+            'yellowZone' => _t('Yellow zone duration'),
+            'timerPolicy' => _t('Timer policy'),
+            'autoSeating' => _t('If automatic seating is allowed'),
+            'isOnline' => _t('If event is online'),
+            'gameDuration' => _t('Game duration in minutes'),
+            'withLeadingDealerGameover' => _t('If game ends when leading dealer wins in last round'),
+            'isTextlog' => null, // not intended to be shown in rule overview
+            'syncStart' => _t('If games should be started simultaneously'),
```
Comment (bogachev-pa) [inline Rheda/src/helpers/Config.php]: `If all the games should be started simultaneously (tournament mode)`
я бы так написал

File: Rheda/src/helpers/Config.php [hunk 2]
```diff
@@ -49,6 +49,57 @@ class Config
...
+            'syncEnd' => _t('If games should be finished simultaneously'),
```
Comment (bogachev-pa) [inline Rheda/src/helpers/Config.php]: `If all the games should be finished simultaneously (tournament mode)`

File: Rheda/src/helpers/Config.php [hunk 3]
```diff
@@ -49,6 +49,57 @@ class Config
...
+            'sortByGames' => _t('If rating table should be sorted by games count first'),
```
Comment (bogachev-pa) [inline Rheda/src/helpers/Config.php]: `by number of games played`? не уверен, как лучше тут

File: Rheda/src/helpers/Config.php [hunk 4]
```diff
@@ -49,6 +49,57 @@ class Config
...
+            'allowPlayerAppend' => _t('If new players are allowed to join in the middle of event'),
+            'useTimer' => _t('If timer should be used'),
+            'usePenalty' => _t('If arbitrary penalties are enabled'),
+            'seriesLength' => _t('If game series are enabled'),
+            'gamesStatus' => null, // not intended to be shown in rule overview
+            'hideResults' => null, // not intended to be shown in rule overview
+            'hideAddReplayButton' => null, // not intended to be shown in rule overview
+            'isPrescripted' => _t('If seating for all games is entered in advance')
```
Comment (bogachev-pa) [inline Rheda/src/helpers/Config.php]: `If even uses fixed seating`, мб так?

**Comment by ctizen** [comment]
Остальное поправил. @Nihisil @bogachev-pa

**Pull request merged**